platform/units.hpp

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include "std/string.hpp"
```

Comment:
OK, as measurement_utils.hpp is in platform now, why not to put this code to measurement_utils.hpp?